README.md

```diff
@@ -61,7 +61,13 @@ You can wire up a context menu like this:
         {label:'Another Thing', icon:'icons/receipt-text.png',    action:function() { alert('clicked 2') } },
         null, /* null can be used to add a separator to the menu items */
         {label:'Blah Blah',     icon:'icons/book-open-list.png',  action:function() { alert('clicked 3') }, isEnabled:function() { return false; } },
-      ]});
+      ],
+      dataTokenLookups: [
+        'name-of-data-attribute-to-retrieve-from-bound-element-and-placed-on-items[0]',
```

Comment:
I don't understand what these dataTokenLookups are for. It appears in the examples and readme, but there's no documentation of how they work.